google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -47,10 +47,52 @@ class Bucket
         attr_accessor :gapi
 
         ##
+        # A `true`/`false` value or a project ID string to be sent as the
```

Comment:
Could you define what it means for `true` to be set? (Use project id defined by default to use `project`)